config/environments/development.rb

```diff
@@ -12,6 +12,9 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
+  # Let's use delayed job in development to process background jobs.
+  config.active_job.queue_adapter = :delayed_job
+
```

Comment:
Why switch from the Rails default of `:async`? 🤔

Reply:
Hmm maybe I'm doing something wrong but when I've started app using foreman jobs have not been processed.

Reply:
I'm not sure if using foreman changes anything (I just use `rails s` in dev), but the default `:async` adapter will run the job in a parallel thread: https://api.rubyonrails.org/classes/ActiveJob/QueueAdapters/AsyncAdapter.html

This setup _should_ cause any deferred jobs to run pretty much as soon as they're queued, in development.